### DISCUSSION TITLE
Deprecation or Removal of AS2 Transport Protocol support in Oxalis

### DISCUSSION THREAD

**Discussion by aaron-kumar** [OxalisCommunity/oxalis#601] [Polls]
As you know, AS4 became mandatory transport protocol according to OpenPeppol guidelines starting 2020-02-01 (https://docs.peppol.eu/edelivery/migration/as2-to-as4-2020/)

It is mandatory that each Access Point must support AS4 mandatory protocol. All participant must be registered in SMP to support mandatory AS4 transport protocol. 

Given condition above, there is plan to either deprecate or remove AS2 protocol support from Oxalis

**Comment by quintensis** [comment]
Hello,

 
Removal of AS2 from the source is fine by us.

 
 
Met vriendelijke groet / Kind regards,
Henk van Koeverden


 
Secoya campus gebouw B

Papendorpseweg 97

3528 BJ Utrecht

W:  www.tradeinterop.com <http://www.tradeinterop.com/> 

 
 
 
 
Van: Arun Kumar ***@***.***> 
Verzonden: donderdag 14 juli 2022 15:36
Aan: OxalisCommunity/oxalis ***@***.***>
CC: Subscribed ***@***.***>
Onderwerp: [OxalisCommunity/oxalis] Deprecation or Removal of AS2 Transport Protocol support in Oxalis (Discussion #601)

 
As you know, AS4 became mandatory transport protocol according to OpenPeppol guidelines starting 2020-02-01 (https://docs.peppol.eu/edelivery/migration/as2-to-as4-2020/ <https://docs.peppol.eu/edelivery/migration/as2-to-as4-2020/> )

It is mandatory that each Access Point must support AS4 mandatory protocol. All participant must be registered in SMP to support mandatory AS4 transport protocol.

Given condition above, there is plan to either deprecate or remove AS2 protocol support from Oxalis

**Comment by aaron-kumar** [comment]
@quintensis : You can just Vote. No read to reply to message.
